behat/tests/features/OrganisationManagement/edit-members.feature

```diff
@@ -0,0 +1,150 @@
+Feature: Organisation admins can edit members of their organisation
+
+  Scenario: Set up organisation and users
+    Given the following organisations exist:
+      | name               | emailIdentifier              | activated |
+      | Malvern Solicitors | @malvern.example             | true      |
+      | Rey Gusciora       | guscioraonline@gmail.example | true      |
+    And the following users exist:
+      | ndr      | deputyType | firstName | lastName | email                        | postCode | activated |
+      | disabled | PROF       | Lavern    | Degroot  | l.degroot@malvern.example    | SW1H 9AJ | true      |
+      | disabled | PROF       | Landon    | Swayzer  | l.swayzer@outsource.example  | SW1H 9AJ | true      |
+      | disabled | PROF       | Rey       | Gusciora | guscioraonline@gmail.example | SW1H 9AJ | true      |
+    And the following users are in the organisations:
+      | userEmail                    | orgName            |
+      | l.degroot@malvern.example    | Malvern Solicitors |
+      | guscioraonline@gmail.example | Rey Gusciora       |
```

Comment:
This reads really well now 👍

Reply:
It's so nice not having to remember what state `behat-user-prof1` is in 🙏

Reply:
This may not be entirely relevant here, but for future, there is a `Given the following court orders exist:` trait method, which when given a `PA` or `PROF` deputy type, will create them an org and add them to it. If org exists, it will add them to existing.